<!doctype html>
<html lang="en">

  <head>
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <meta name="description" content="">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, user-scalable=no">

    <title>Crown Lab at Case Western Reserve University</title>

    <link rel="stylesheet" href="css/main.css">
    <link rel="stylesheet" href="https://use.typekit.net/ulq8tbj.css">
    <link rel="icon" type="image/png" href="images/favicon-32.png" sizes="32x32">
    <link rel="icon" type="image/png" href="images/favicon-16.png" sizes="16x16">
  </head>

  <body>
      <div class="cl-content__grid">

        <div class="cl-content__header">
          <div class="cl-content-wrapper">
            <header class="cl-header">

              <a href="index.html" class="cl-link--no-decoration">
                <div class="cl-logo">
                    <span class="cl-logo__image">
                      <img class="cl-image" src="images/logo.svg" alt="Logo for Crown Lab" />
                    </span>
                    <span class="cl-logo__text">CROWN LAB</span>
                </div>
              </a>

              <nav class="cl-header-nav">
                <ul class="cl-header-nav__list">
                  <li class="cl-header-nav__item"><a class="cl-header-nav__anchor" href="news.html">News</a></li>
                  <li class="cl-header-nav__item"><a class="cl-header-nav__anchor" href="research.html">Research</a></li>
                  <li class="cl-header-nav__item"><a class="cl-header-nav__anchor" href="people.html">People</a></li>
                  <li class="cl-header-nav__item"><a class="cl-header-nav__anchor" href="join.html">Join Us</a></li>
                  <li class="cl-header-nav__item"><a class="cl-header-nav__anchor" href="contact.html">Contact</a></li>
                </ul>
              </nav>
            </header>
          </div>
        </div>

        <div class="cl-content__main">
          <div class="cl-content-wrapper">
            <div class="cl-main cl-main--two-column">
              <aside class="cl-content-aside">
                <div class="cl-content-aside__header">
                  SECTIONS
                </div>
                <div class="cl-content-aside__menu">
                  <ul class="cl-content-aside__list">
                    <li class="cl-content-aside__item">
                      <a href="#overview" class="cl-content-aside__anchor">Overview</a>
                    </li>
                    <li class="cl-content-aside__item">
                      <a href="#inversions" class="cl-content-aside__anchor">Inversions</a>
                    </li>
                    <li class="cl-content-aside__item">
                      <a href="#repair" class="cl-content-aside__anchor">DSB Repair Mechanics</a>
                    </li>
                  </ul>
                </div>
              </aside>
              <div class="cl-main__content">

                <h1 class="cl-main__header"><a id="overview"></a>Overview</h1>
                <section class="cl-main__section">
                  <p class="cl-main__paragraph">
                    <img src="images/research/n2n.png" class="cl-image cl-image--left cl-image--med" />
                    Sexually reproducing species face a fundamental math problem: each individual has two copies of the genome, one maternal and one paternal. If each individual passes on both copies of the genome, their progeny will have four copies; the next generation would then have eight copies, and so on. To solve this math problem, sexually reproducing organisms pass on just one copy of the genome using a specialized cell division process called meiosis.
                  </p>
                  <p class="cl-main__paragraph">
                    <img src="images/research/dsb.png" class="cl-image cl-image--right cl-image--lg" />
                    Unlike mitosis, in which cells divide only once to produce identical diploid daughter cells, meiosis involves two rounds of cell division. After genome replication, homologous chromosomes find each other in the nucleus and separate during the first meiotic division. Sister chromatids then separate in the second meiotic division, resulting in four genetically unique haploid gametes, each bearing half the chromosome complement.
                  </p>
                  <p class="cl-main__paragraph">
                    The cell uses a series of complex processes to ensure homologous chromosomes segregate before sister chromatids. Chief among these is the formation of crossovers, which are the reciprocal exchange of DNA between homologous chromosomes. Crossovers ensure homologous chromosomes are paired before the first division happens. They are also one of the mechanisms that generate genetic diversity among the four haploid products (we can have a heated discussion over which one of these functions in most important).
                  </p>
                  <p class="cl-main__paragraph">
                    <img src="images/research/co-diagram.png" class="cl-image cl-image--left cl-image--xl" />
                    Crossovers form by using homologous recombination to repair programed DNA double-stranded breaks. More breaks are made than crossovers, however, and those breaks that are not repaired as crossovers are repaired as noncrossovers (gene conversions). The distribution of crossovers along a chromosome occurs in a stereotyped pattern that is shaped by both crossover-promoting and crossover-suppressing forces, collectively referred to as crossover patterning mechanisms. Since only crossovers made in specific parts of the chromosome are effective for chromosome segregation, the choice of repair outcome is a critical regulatory point in establishing the distribution of crossovers across the genome.
                  </p>
                </section>

                <h1 class="cl-main__header"><a id="inversions"></a>Inversions</h1>
                <section class="cl-main__section">
                  <p class="cl-main__paragraph">
                    Chromosome inversions are naturally occurring structural aberrations that, when heterozygous, disrupt the recombination landscape by suppressing crossing over. The crossover suppression is due to two distinct mechanisms. First, single crossovers that occur within an inversion result in structurally abnormal chromosomes that are generally lethal and never recovered in the progeny. Second, inversion breakpoints prevent crossovers from forming in the immediate vicinity. Intriguingly, in Drosophila species, heterozygous inversions on one chromosome trigger a dramatic increase in crossing over on other structurally linear chromosomes in the genome. This phenomenon is referred to as the interchromosomal effect and was studied at the phenomenological level for decades, but very little progress has been made in understanding its molecular basis.
                  </p>
                  <p class="cl-main__paragraph">
                      We’ve shown that heterozygous inversions change the recombination landscape by altering repair outcome in favor of crossovers on the structurally linear chromosomes but in favor of noncrossovers around inversion breakpoints. We are currently using whole genome sequencing approaches to address the mechanism by which heterozygous inversions alter the crossover/noncrossover decision to rearrange the recombination landscape.
                  </p>
                  <cite class="cl-main__citation">
                      <b>Crown, K.N.*</b>, D.E. Miller*, J. Sekelsky, R.S. Hawley, Local Inversion Heterozygosity Alters Recombination Throughout the Genome. Current Biology, in press.
                  </cite>
                </section>

                <h1 class="cl-main__header"><a id="repair"></a>DSB Repair Mechanics</h1>
                <section class="cl-main__section">
                  <p class="cl-main__paragraph">
                    <img src="images/research/co-repair-diagram.png" class="cl-image cl-image--left cl-image--xxxl" />
                    The current working model of meiotic recombination based on work in budding yeast is that double-strand breaks repaired as noncrossovers use a simple synthesis-dependent strand annealing mechanism, whereas double-strand breaks repaired as crossovers proceed through a more complicated intermediate called a double Holliday junction. These are genetically separable pathways, with evidence that noncrossovers form early and independently of crossovers.
                  </p>
                  <p class="cl-main__paragraph">
                      We have shown that, in Drosophila, two classes of noncrossovers form from distinct structural intermediates. One class appears to use canonical synthesis-dependent strand annealing, while the other class comes from an intermediate where both ends of the double-strand break interact with the homologous chromosome (we’ve termed this the two-end engagement intermediate). Additionally, we’ve proposed that crossovers form through nicking of the same two-end engagement intermediate.
                  </p>
                  <p class="cl-main__paragraph">
                    <img src="images/research/flow.png" class="cl-image cl-image--left cl-image--xl" />
                      Since crossovers and noncrossovers potentially come from differential resolution of the same structural intermediate in Drosophila, the decision to repair a break as a crossover or a noncrossover may occur late during recombination. A late repair decision may allow for flexibility during prophase when crossover patterning mechanisms are active. We are testing the hypothesis that flexibility and a late crossover/noncrossover decision are defining characteristics of meiotic recombination in Drosophila, and that higher-order crossover patterning mechanisms rely on that flexibility to establish the correct crossover distribution.
                  </p>
                  <cite class="cl-main__citation">
                    <b>Crown, K.N.</b>, S. McMahan, and J. Sekelsky, <em>Eliminating Both Canonical and Short-Patch Mismatch Repair in Drosophila melanogaster Suggests a New Meiotic Recombination Model.</em> PLoS Genetics, 2014. 10(9): p. e1004583. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=PMC4154643">PMC4154643.</a> | <a href="http://journals.plos.org/plosgenetics/article/file?id=10.1371/journal.pgen.1004583&type=printable">PDF.</a>
                  </cite>
                </section>

              </div>
            </div>
          </div>
        </div>

        <div class="cl-content__footer">
          <div class="cl-content-wrapper">
            <footer class="cl-footer">
              <div class="cl-footer__copy">
                <span class="cl-footer__copy-main">© 2018 Crown Laboratory</span>
                <span class="cl-footer__copy-sub">Case Western Reserve University</span>
              </div>
              <div class="cl-footer__social">
                <div class="cl-footer__social-text">Follow us on</div>
                <div class="cl-footer__social-icon"><a href="#"><img class="cl-image cl-image--inline cl-icon-size--med" src="images/twitter.svg" alt="logo for twitter" /></a></div>
              </div>
            </footer>
          </div>
        </div>

      </div>
  </body>
</html>